index.js

```diff
@@ -12,6 +35,63 @@ modalClose.addEventListener('click', (e) => {
 
 window.addEventListener('click', (e) => {
     if (e.target == index) {
-        index.style.display = "none";
+        index.style.display = "none"
     }
 })
+
+let allKeys = document.querySelectorAll('.keypadKey')
+let wordInput = document.getElementById('wordInput')
+
+allKeys.forEach((key) => {
+    key.addEventListener('click', () => {
+        if (key.dataset.letter === 'delete') {
+            wordInput.value = wordInput.value.substring(0, wordInput.value.length - 1)
+        } else {
+            wordInput.value += key.dataset.letter
+        }
+        enableEnterButton(wordInput.value.length)
+    })
+})
+
+document.getElementById("enterButton").addEventListener('click', (e) => {
+
+    e.preventDefault();
+    guessedWord = wordInput.value
+    console.log(guessedWord)
+    console.log(randomWord)
+
+     if (guessedWord === randomWord) {
+        matchResult = true
+    } else {
+        matchResult = false
+    }
+    outcomeOutput(matchResult, guessedWord)
+
+})
+
+function outcomeOutput(bool, inputWord) {
+    document.querySelector('.textInput').style.display = 'none'
+    document.querySelector('.submitFormButton').style.display = 'none'
+    document.querySelector('form').style.justifyContent = 'center'
+    if(bool) {
+        let correctTag = document.createElement('p')
+        let correctText = document.createTextNode(inputWord + ' was correct')
+        correctTag.appendChild(correctText)
+
+        document.querySelector('form').appendChild(correctTag)
+    } else {
+        let incorrectTag = document.createElement('p')
```

Comment:
this could be done a bit more dryly - so you dont have to createElement twice, if you create it once outside the if statement it would be neater

Reply:
(and just set the createTextNode inside the if statement)